assets/scss/modules/_forms.scss

```diff
@@ -0,0 +1,330 @@
+form {
+  text-align: center;
+  font-family: 'proxima-nova', sans-serif;
+
+  .rgpd {
+    margin: 30px 0 10px;
+
+    p {
+      text-align: left;
+      font-size: 14px;
+      margin: 5px 0;
+    }
+  }
+
+
+  .intl-tel-input.separate-dial-code .selected-dial-code {
+    font-size: 14px;
+  }
+
+  .intl-tel-input .country-list {
+    max-width: 306px;
+    overflow-x: hidden;
+
+    .country {
+      display: flex;
+      align-items: center;
+
+      &:after {
+        content: '';
+        position: absolute;
+        height: 35px;
+        width: 1px;
+        left: 33%;
+        border-left: 1px solid #B4B1A5;
```

Comment:
Could you please check that colors are defined into vars, as in the rest of the library? for clarity and consistensy.